README.rst

```diff
@@ -69,12 +69,12 @@ Zgen_
 
 oh-my-zsh_
 
-Copy this repository to ``$ZSH/custom/plugins``, where ``$ZSH`` is the
-root directory of oh-my-zsh:
+Copy this repository to ``$ZSH_CUSTOM/custom/plugins``, where ``$ZSH_CUSTOM`` 
+is the directory with custom plugins of oh-my-zsh:
 
 ::
 
-    git clone https://github.com/MichaelAquilina/zsh-you-should-use.git $ZSH/custom/plugins/you-should-use
+    git clone https://github.com/MichaelAquilina/zsh-you-should-use.git $ZSH_CUSTOM/plugins/you-should-use
 
```

Comment:
not sure if maybe you forgot to push or if I was not clear, but could we include the link you provided in your previous comment here?

Maybe something like "See more details about custom plugins [here](link_url)"

Reply:
I got, what you mean. Thought that you about that I forgot to change that path near link after git clone :)

Reply:
Thanks! The link seems broken though (Check https://github.com/MichaelAquilina/zsh-you-should-use/blob/86f40c68c41aec519f059d22316aa892bc8d6808/README.rst)

Reply:
Just remembered why. The README uses restructured text. See this page for how to add links http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks